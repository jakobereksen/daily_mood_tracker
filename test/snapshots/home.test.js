const { app, request } = require("../commonJest");

it("renders home correctly", async (done) => {

	const data = await request(app).get("/");

	expect(data.text).toMatchSnapshot();
	done();
/* 	request(app).get("/").then((response) => {
		expect(response.text).toMatchSnapshot();
		done();	
	}); */
});
