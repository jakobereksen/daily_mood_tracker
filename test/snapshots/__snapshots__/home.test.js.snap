<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Daily Mood Tracker</title>
    <link rel="stylesheet" type="text/css"   href="/css/dailymoodtracker.css">
    <link rel="stylesheet" type="text/css"   href="/css/bootstrap.css">
  </head>
  <body>

    <header>
      <h1 class="logo">Daily Mood Tracker</h1>

      <nav>
          <div class="nav-elements">
            <a class="nav-element" href="questionnaire">Neuer Eintrag</a>
            <a class="nav-element" href="statistics">Mein Stimmungsverlauf</a>
            <a class="nav-element">Einstellungen</a>
            <a class="nav-element">Abmelden</a>
          </div>
      </nav>

    </header>

    <body>

    <h1> Hello and welcome! <%= name %> </h1>

    <%- include('partials/notification'); %>

    <div class="footer">FOOTER</div>
  </body>
</html>